app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
+require "#{Rails.root}/lib/helpers/api_helpers"
```

Comment:
Does anyone have a preference between explicitly requiring files in `lib` vs. autoloading them by default?

Reply:
For me it depends on the size of the file, how many files use it, and the load time.  I am also curious if anyone else has opinions.